src/som/vmobjects/SSymbol.java

```diff
@@ -41,10 +40,9 @@
   public SSymbol(final String value) {
     string = value;
     numberOfSignatureArguments = determineNumberOfSignatureArguments();
-    if (VmSettings.KOMPOS_TRACING || VmSettings.ACTOR_TRACING
-        || VmSettings.TRACK_SNAPSHOT_ENTITIES) {
+    if (VmSettings.KOMPOS_TRACING || VmSettings.TRACK_SNAPSHOT_ENTITIES) {
       symbolId = (short) idGenerator.getAndIncrement();
-      TracingBackend.logSymbol(this);
```

Comment:
Hmm. Does this work for Kompos?
Does the logSymbol still need to be done for it?

Reply:
I restored the call. Seemed still necessary.